### PR TITLE
Fixed minor documentation issues

### PR DESCRIPTION
**Pull request by pavangudiwada**
* Mentioned Kubernetes secret configuration guide in Installation page.
* Added a warning about “Alerts won't show up” notification on Robusta UI.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -113,6 +113,11 @@ Standard Installation
 
     helm install robusta robusta/robusta -f ./generated_values.yaml --set clusterName=<YOUR_CLUSTER_NAME> # --set isSmallCluster=true
 
+    
+.. note::
+
+      You can provide configuration values as Kubernete secrets. See `Configuration secrets <https://docs.robusta.dev/master/user-guide/configuration-secrets.html>`_ guide.
```
Comment (arikalon1) [inline docs/installation.rst:119]: I would change to:
Sensitive configuration values can be stored in Kubernetes secrets. See `Configuration secrets <https://docs.robusta.dev/master/user-guide/configuration-secrets.html>`_ guide.

File: docs/user-guide/alert-manager.rst [hunk 2]
```diff
@@ -52,6 +52,11 @@ After you configured the ``receiver`` and ``route``, you can test it works prope
 
 You should see the demo alert, in the Robusta UI, Slack, or any other configured ``sink`` within a few minutes
 
+.. admonition:: "Alerts won't show up" UI notification
+    :class: warning
+
+    "Alerts won't show up because Prometheus is not connected to Robusta" will always be there even if the connection is configured correctly, until the first alert has been raised.
```
Comment (arikalon1) [inline docs/user-guide/alert-manager.rst:58]: I would try to make it shorter. How about:
The notification is displayed until the first alert is sent from AlertManager to Robusta

**Review by arikalon1 — APPROVED** [review]
Looks good
Left 2 minor comments
Feel free to merge

File: docs/installation.rst (hunk 1 above)
Comment (pavangudiwada) [reply docs/installation.rst]: Fixed.

File: docs/user-guide/alert-manager.rst (hunk 2 above)
Comment (pavangudiwada) [reply docs/user-guide/alert-manager.rst]: Made the specified changes.

**Pull request merged**